README.md

```diff
@@ -1,24 +1,20 @@
 
 # VyOS Collection
 
-[![Codecov](https://codecov.io/gh/ansible-collections/vyos.vyos/branch/main/graph/badge.svg)](https://codecov.io/gh/ansible-collections/vyos.vyos)
-[![CI](https://github.com/ansible-collections/vyos.vyos/actions/workflows/tests.yml/badge.svg?branch=main&event=schedule)](https://github.com/ansible-collections/vyos.vyos/actions/workflows/tests.yml)
+[![Codecov](https://codecov.io/gh/vyos/vyos.vyos/branch/main/graph/badge.svg)](https://codecov.io/gh/ansible-collections/vyos.vyos)
+[![CI](https://github.com/vyos/vyos.vyos/actions/workflows/tests.yml/badge.svg?branch=main&event=schedule)](https://github.com/vyos/vyos.vyos/actions/workflows/tests.yml)
 
 The Ansible VyOS collection includes a variety of Ansible content to help automate the management of VyOS network appliances.
 
-This collection has been tested against VyOS 1.1.8 (helium).
+This collection has been tested against VyOS 1.3.8 and 1.4.1.
```

Comment:
I guess it was also tested against 1.5? If not, do you want me to test it?

Reply:
It has been, although the lack of an actual 1.5 release makes me hesitant to declare testing against it, since 1.5 could make changes to be incompatible.

Reply:
I went ahead and upadated it, thanks.